flatten/flatten_spec.go

```diff
@@ -457,3 +465,45 @@ func flattenAzure(in azAlpha1.InfrastructureConfig) []interface{} {
 
 	return []interface{}{att}
 }
+
+func flattenAws(in awsAlpha1.InfrastructureConfig) []interface{} {
+	att := make(map[string]interface{})
+	net := make(map[string]interface{})
+	vpc := make(map[string]interface{})
+
+	// if in.EnableECRAccess != nil {
```

Comment:
We can comment this in I suppose since its used during expand_spec.

Reply:
PR updated with latest commit.